interfaces/builtin/network_control.go

```diff
@@ -280,6 +284,17 @@ var networkControlConnectedPlugUDev = []string{
 	`KERNEL=="tun"`,
 }
 
+var networkControlConnectedPlugMount = []osutil.MountEntry{{
+	Name:    "/var/lib/snapd/hostfs/var/lib/dhcp",
+	Dir:     "/var/lib/dhcp",
+	Options: []string{"rw", "bind"},
+}}
+
+var networkControlConnectedPlugUpdateNSAppArmor = `
+mount options=(rw bind) /var/lib/snapd/hostfs/var/lib/dhcp/ -> /var/lib/dhcp/,
+umount /var/lib/dhcp/,
```

Comment:
Got this locally:
```
update.go:85: cannot change mount namespace according to change mount (/var/lib/snapd/hostfs/var/lib/dhcp /var/lib/dhcp none rw,bind 0 0): cannot open directory "/var/lib": p
ermission denied 
```

looks like we may need:
```
/var/lib/ r,
```

Reply:
Oh, that's interesting, your system did not have `/var/lib`? In either case you are spot on. I'll correct this.

Reply:
It has /var/lib, but no /var/lib/dhcp

Reply:
Hmm, then something weird is going on. How can the spread test I added pass? On at least three systems we don't have /var/lib/dhcp.

Reply:
Done, ouch, we needed quite a bit more.